h2o-k8s/src/main/java/water/k8s/KubernetesEmbeddedConfigProvider.java

```diff
@@ -65,7 +65,7 @@ public void init() {
         if (!runningOnKubernetes) {
             return; // Do not initialize any configuration if H2O is not running in K8S-spawned container.
         }
-
+        System.setProperty(H2OHttpViewImpl.DISABLE_NON_LEADER_API, "true");
```

Comment:
- There is also the option to manipulate some state directly, and not indirectly via a system property.
- Not sure if embedded config should manipulate any internal state directly. Probably not. One way to solve this is to introduce yet another overridable method in EmbeddedConfig to override/set system properties and args alike. At the moment, this would have to be done just for this functionality.
- The only user of thins functionality is now Kubernetes, yet it seems like a good idea to expose this setting as a sysprop for anyone to set it in any environment, if required.